scalardb-test/src/main/java/kelpie/scalardb/transfer/TransferProcessorWithRetry.java

```diff
@@ -0,0 +1,108 @@
+package kelpie.scalardb.transfer;
+
+import com.scalar.db.api.DistributedTransaction;
+import com.scalar.db.api.DistributedTransactionManager;
+import com.scalar.db.api.Get;
+import com.scalar.db.api.Put;
+import com.scalar.db.api.Result;
+import com.scalar.db.exception.transaction.CommitConflictException;
+import com.scalar.db.exception.transaction.CrudConflictException;
+import com.scalar.db.exception.transaction.RollbackException;
+import com.scalar.kelpie.config.Config;
+import com.scalar.kelpie.modules.TimeBasedProcessor;
+import io.github.resilience4j.core.IntervalFunction;
+import io.github.resilience4j.retry.Retry;
+import io.github.resilience4j.retry.RetryConfig;
+import java.util.Optional;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.atomic.LongAdder;
+import javax.json.Json;
+
+public class TransferProcessorWithRetry extends TimeBasedProcessor {
+
+  private static final int INITIAL_INTERNAL_MILLS = 10;
+  private static final double MULTIPLIER = 2.0;
+  private static final int MAX_RETRIES = 10;
+
+  private final DistributedTransactionManager manager;
+  private final int numAccounts;
```

Comment:
Why not using `long` as well as `config.getUserLong`?

Reply:
We need to do the same thing for other classes. So we should do that in another PR. Thanks.